### PR TITLE
Add Discord PTB profile

### PR DESCRIPTION
**Pull request by Neotamandua**
Same profile behavior as Discord Canary, but for PTB

### CODE REVIEW

File: etc/profile-a-l/discord-ptb.profile [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Firejail profile for discord-ptb   
+# This file is overwritten after every install/update
+# Persistent local customizations
+include discord-ptb.local   
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}/.config/discordptb
```
Comment (rusty-snake) [inline etc/profile-a-l/discord-ptb.profile:8]: Never blacklisted, add a blacklist to disable-programs.

File: etc/profile-a-l/DiscordPTB.profile [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Firejail profile for DiscordPTB
```
Comment (rusty-snake) [inline etc/profile-a-l/DiscordPTB.profile:1]: firecfg?

**Review by rusty-snake — CHANGES_REQUESTED** [review]

File: etc/profile-a-l/discord-ptb.profile (hunk 1 above)
Comment (Neotamandua) [reply etc/profile-a-l/discord-ptb.profile:8]: I am not sure what you mean by this. To me it looks the same as https://github.com/netblue30/firejail/blob/master/etc/profile-a-l/discord-canary.profile but for ptb

Comment (rusty-snake) [reply etc/profile-a-l/discord-ptb.profile:8]: But there is no blacklist of it unlike for discordcanary:
https://github.com/netblue30/firejail/blob/ddb685c1ec66f9164e0721b0d8bb5ccb1a75e22d/etc/inc/disable-programs.inc#L414

File: src/firecfg/firecfg.config [hunk 3]
```diff
@@ -11,6 +11,7 @@ Cryptocat
 Cyberfox
 Discord
 DiscordCanary
+DiscordPTB
```
Comment (kmk3) [inline src/firecfg/firecfg.config:14]: Since this PR adds discord-ptb.profile, add `discord-ptb` in this file as well.

**Review by rusty-snake — APPROVED** [review]

**Review by kmk3 — APPROVED** [review]

**Comment by netblue30** [comment]
merged, thanks!

**Pull request merged**